distribution/lib/Standard/Base/0.0.0-dev/src/Data/Index_Sub_Range.enso

```diff
@@ -67,6 +74,22 @@ type Index_Sub_Range
         Index_Sub_Range.Sample count _ -> "Sample " + count.to_display_text
         Index_Sub_Range.Every step first -> "Every " + step.to_display_text + (if first == 0 then "" else " from " + first.to_display_text)
 
+    ## PRIVATE
+    default_options : Vector Option
+    default_options =
+        o1 = Option "First" "(Index_Sub_Range.First 1)"
+        o2 = Option "Last" "(Index_Sub_Range.Last 1)"
+        o3 = Option "While" "(Index_Sub_Range.While (x-> False))"
+        o4 = Option "By_Index" "(Index_Sub_Range.By_Index [0])"
+        o5 = Option "Sample" "(Index_Sub_Range.Sample 10)"
+        o6 = Option "Every" "(Index_Sub_Range.Every 2)"
+        [o1, o2, o3, o4, o5, o6]
```

Comment:
Should we include a "range" here?

Reply:
Yeah, probably right. I will add one. What default value do you suggest? Full range? first element?